DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mpoxseir
 Title: Stochastic compartmental model of mpox transmission
-Version: 0.0.1
+Version: 0.0.2
 Authors@R: c(person("Lilith", "Whittles", role = c("aut", "cre"),
                     email = "l.whittles@imperial.ac.uk"),
              person("Ruth", "McCabe", role = c("aut")),
```

Comment:
My only comment is that both @edknock and @cwhittaker1000 should definitely appear here